### PR TITLE
add confirmation view and gateway

### PR DESCRIPTION
**Pull request by elena-vi**

### CODE REVIEW

File: flow.js [hunk 1]
```diff
@@ -99,13 +99,11 @@ class Flow {
...
       //   {condition: 'confirm-and-report', nextStep: 'confirmation'}
       // ]},
-      'confirmation': {prevStep:'summary', nextStep: [
-        {condition: 'report-another-issue', nextStep: 'index'},
-        {condition: 'request-confirmation', nextStep: 'confirmation-template'}
-      ]}
     }
   };
   nextStep (step, state, prevStep) {
+    console.log('nextStep')
```
Comment (Bhavesh-Patel) [inline flow.js:105]: Are these `console.log`s required? If not, please delete.

File: tests/cypress/integration/reportRepair/confirmation.spec.js [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+  it('Does not display the back button', () => {
+    cy.get('.govuk-back-link').should('not.exist')
+  });
+
+  context('when user sends confirmation via email', ()=>{
+    it('Displays where the confirmation was sent to', () => {
+      cy.contains('We have sent a confirmation to ' + email);
+    });
+
+    // it('checks request',  () =>{
```
Comment (Bhavesh-Patel) [inline tests/cypress/integration/reportRepair/confirmation.spec.js:110]: Should this test be uncommented or deleted?

File: flow.js (hunk 1 above)
Comment (elena-vi) [reply flow.js:105]: whoops my bad

**Comment by Bhavesh-Patel** [comment]
If a photo isn't provided, then the repair request isn't saved. The manifests by no repair number being displayed upon submission of the repair request.

**Comment by elena-vi** [comment]
The bug raised was fixed in https://github.com/City-of-Lincoln-Council/housing-repairs-online-api/pull/19

**Review by Bhavesh-Patel — APPROVED** [review]
Added small issue [comment](https://github.com/City-of-Lincoln-Council/housing-repairs-online-frontend/pull/40#pullrequestreview-854473368).

Please fix prior to merging, approving now though.

**Pull request merged**